pkg/virt-launcher/virtwrap/agent-poller/agent_parser_test.go

```diff
@@ -27,174 +27,7 @@ import (
 )
 
 var _ = Describe("Qemu agent poller", func() {
-	Context("recieving a reply from the agent", func() {
```

Comment:
Should we keep these tests for the parsers which have been changed to converters and moved to agent_poller.go?

Reply:
I don't think so. These are private functions that are expected to be already tested here:
```
It("should store the retrieved guest info when requested", func() {
...
}
```
This test already testing the converted info.

Reply:
ACK